multisite/models.py

```diff
@@ -226,7 +213,7 @@ def validate_unique(self, exclude=None):
             field_error = self.unique_error_message(self.__class__,
                                                     (field_name,))
             if field_name not in errors or \
-               field_error not in errors[field_name]:
+               str(field_error) not in [str(err) for err in errors[field_name]]:
```

Comment:
@mghughes I've modified this because even when the unique validation error was in errors[field_name], ```field_error not in errors[field_name]``` was True, so the validation error was showing up twice in the form.  Not sure why...

```
ipdb> field_error
ValidationError([u'Alias with this Domain name already exists.'])
ipdb> errors[field_name]
[ValidationError([u'Alias with this Domain name already exists.'])]
ipdb> field_error not in errors[field_name]
True
ipdb> field_error == errors[field_name][0]
False
ipdb> type(field_error)
<class 'django.core.exceptions.ValidationError'>
ipdb> [type(err) for err in errors[field_name]]
[<class 'django.core.exceptions.ValidationError'>]
ipdb> str(field_error) == str(errors[field_name][0])
True
ipdb> errors
{'domain': [ValidationError([u'Alias with this Domain name already exists.']), ValidationError([u'Alias with this Domain name already exists.'])]}
```